styleguide/derek/examples/_homepage.html

```diff
@@ -223,40 +231,48 @@ <h2 class="right">FANATICAL SUPPORT WhereVER YOU ARE, DaY OR NIGHT, WE HAVE YOU
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/yeti-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/rip-curl-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/11/six-flags-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/xero-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/kendra-scott-logo.png</li>
-          <li>http://stories.rackspace.com/wp-content/uploads/2015/09/gsdm-logo.png</li>
-          <li></li>
-        </ul>
+  <div class="container-fluid half-padding bg-blue-gray">
+    <div class="container customer_stories_logos">
+      <div class="row">
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/underarmor.png" alt="">
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/yeti-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/rip-curl-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/six-flags-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/ksLogo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/razorfish-logo.png"/>
+        </div>
       </div>
     </div>
   </div>
 
     
    
   <!-- Contact River -->
-  <div class="container-fluid standard-padding bg-dark-gray">
+  <div class="container-fluid standard-padding bg-light-gray">
     <div class="container">
       <div class="row half-padding">
         <div class="col-md-8 center-block">
-          <h2 class="white center">Get Free Advice From Our Experts</h2>
-          <h5 class="white center">We've got your back 24x7x365.</h5>
+          <h2 class="center">Get Free Advice From Our Experts</h2>
+          <h5 class="center">We've got your back 24x7x365.</h5>
           <div class="col-sm-2 center-block">
-            <i class="rsweb gen-support white center"></i>
+            <i class="rsweb gen-support center"></i>
           </div>
         </div>
       </div>
           <div class="row">
             <div class="col-md-6">
-              <p class="white">Our helpful and always ready experts are a click or call away. <br/>Anytime. Anywhere.</p>
+              <p class="">Our helpful and always ready experts are a click or call away. <br/>Anytime. Anywhere.</p>
```

Comment:
I'm not sure if its going to be white or a dark background or not yet. The homepage is in progress.